xtas/tasks/_nl_conll_ner.py

```diff
@@ -31,12 +31,12 @@ def _download_training_data():
 
     Returns an iterable over the lines of the concatenated dataset.
     """
-    return (ln for part in ["train", "testa", "testb"]
```

Comment:
In that case, then so is downloading it in the first place, unless they have an unusual license. I'm planning to do a legal analysis of xtas anyway, so let's leave this for now and we'll figure it out as part of that.